routes/auth.js

```diff
@@ -90,8 +104,11 @@ async function githubAuth(req, res) {
     },
   })
 
-  let userId
-  if (userResp.data) userId = userResp.data.login
+  const githubId = userResp.data && userResp.data.login
+
+  // Create user if does not exists. Set last logged in to current time.
+  const user = await identityServices.usersService.login(githubId)
+  if (!user) throw Error("Failed to create user")
```

Comment:
To confirm: will githubId ever be false? -> will cause bug if false because we will attempt to create a new user with githubId `false`